scripts/onnx/test.sh

```diff
@@ -57,8 +57,10 @@ pytest "${args[@]}" \
 # onnxruntime only support py3
 # "Python.h" not found in py2, needed by TorchScript custom op compilation.
 if [[ "$BUILD_ENVIRONMENT" == *py3* ]]; then
-  pytest "${args[@]}" "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py"
-  pytest "${args[@]}" "$top_dir/test/onnx/test_custom_ops.py"
-  pytest "${args[@]}" "$top_dir/test/onnx/test_models_onnxruntime.py"
+  pytest "${args[@]}" \
+    -k \
+    'not (TestOperators and test_full_like) and not (TestOperators and test_zeros_like) and not (TestOperators and test_ones_like) and not (TestModels and test_vgg16) and not (TestModels and test_vgg16_bn) and not (TestModels and test_vgg19) and not (TestModels and test_vgg19_bn)' \
```

Comment:
This branch is complementary to py2's tests.

Nit: we may create a common variable to store the skipped tests? So later, we don't need to change two places if we want to skip more things.